ts/Series/Spline/SplineSeries.ts

```diff
@@ -175,58 +175,77 @@ class SplineSeries extends LineSeries {
+                    )
+                    .attr({
+                        stroke: 'red',
+                        'stroke-width': 2,
+                        fill: 'none',
+                        zIndex: 9
+                    })
+                    .add();
+                this.chart.renderer
+                    .path([['M', leftContX + this.chart.plotLeft,
+                        leftContY + this.chart.plotTop
+                    ], ['L', plotX + this.chart.plotLeft,
+                        plotY + this.chart.plotTop
+                    ]])
+                    .attr({
+                        stroke: 'red',
+                        'stroke-width': 2,
+                        zIndex: 9
+                    })
+                    .add();
+            }
+            if (rightContX) {
+                this.chart.renderer
+                    .circle(
+                        rightContX + this.chart.plotLeft,
+                        rightContY + this.chart.plotTop,
+                        2
+                    )
+                    .attr({
+                        stroke: 'green',
+                        'stroke-width': 2,
+                        fill: 'none',
+                        zIndex: 9
+                    })
+                    .add();
+                this.chart.renderer
+                    .path([[
+                        'M', rightContX + this.chart.plotLeft,
+                        rightContY + this.chart.plotTop
+                    ], [
+                        'L', plotX + this.chart.plotLeft,
+                        plotY + this.chart.plotTop
+                    ]])
+                    .attr({
+                        stroke: 'green',
+                        'stroke-width': 2,
+                        zIndex: 9
+                    })
+                    .add();
+            }
+            // */
 
-        }
+            point.leftCont = {
+                plotX: leftContX,
+                plotY: leftContY
+            };
 
-        // Visualize control points for debugging
-        /*
-        if (leftContX) {
-            this.chart.renderer.circle(
-                    leftContX + this.chart.plotLeft,
-                    leftContY + this.chart.plotTop,
-                    2
-                )
-                .attr({
-                    stroke: 'red',
-                    'stroke-width': 2,
-                    fill: 'none',
-                    zIndex: 9
-                })
-                .add();
-            this.chart.renderer.path(['M', leftContX + this.chart.plotLeft,
-                leftContY + this.chart.plotTop,
-                'L', plotX + this.chart.plotLeft, plotY + this.chart.plotTop])
-                .attr({
-                    stroke: 'red',
-                    'stroke-width': 2,
-                    zIndex: 9
-                })
-                .add();
+            point.rightCont = {
+                plotX: rightContX,
+                plotY: rightContY
+            };
```

Comment:
There's always a balance between readable and compact code, but since the concept of using tuples for coordinates is so common, I suggest we use `[x, y]` here. Also, to avoid two so similar properties on the point, I think we should just add one. Besides, the words _left_ and _right_ only make sense on a non-reversed, horizontal axis. On a vertical axis, or a reversed one, it is not semantically correct. So I suggest something like this:

```js
point.controlPoints = {
    low: [leftContX, leftContY],
    high: [rightContX, rightContY]
}
```